docs/user/kubevirt/install_ipi.md

```diff
@@ -0,0 +1,162 @@
+>>> Should be following to this structure (remove `{}`):<br/>
+>>> ```console
+>>>     kubevirt:
+>>>       cpu: 4
+>>>       memory: 16G
+>>>       storageSize: 120Gi
+>>> ``` 
+>>> Set different values for compute machines (workers) and for controlplan machines (masters):
+>>>- [workers](./install-config.yaml#L7)
+>>>- [masters](./install-config.yaml#L13)
+>- Change the [workerReplicas](./install-config.yaml#L8) - In case you need copute machines replica different than 3
+>- Change the [masterReplicas](./install-config.yaml#L14) - In case you need controlplan machines replica different than 3
+>- Change the [Machine Network CIDR](./install-config.yaml#L23) to the network range defined in the Network Attachment Definition
+>- Set the [Platform](./install-config.yaml#L27) values, according to the values in the Infra cluster
+>>> Should be following to this structure:<br/>
+>>> ```console
+>>>     kubevirt:
+>>>       apiVIP: x.x.x.y
+>>>       ingressVIP: x.x.x.z
+>>>       namespace: tenantcluster namespace
+>>> ```
+>>> |Value name                  |Value type                    |Required|Description   |
+>>> |---                         |---                           |---     |---|
+>>> |namespace                   |string                        |Yes     |The namespace in the infra cluster, which the control plane (master vms) and the compute (worker vms) are installed in   |
+>>> |storageClass                |string                        |No      |The Storage Class used in the infra cluster   |
+>>> |networkName                 |string                        |Yes     |The target network of all the network interfaces of the nodes   |
+>>> |apiVIP                      |IPV4                          |Yes     |The virtual IP address for the api endpoint   |
+>>> |ingressVIP                  |IPV4                          |Yes     |An external IP which routes to the default ingress controller   |
+>>> |persistentVolumeAccessMode  |[ReadWriteMany,ReadWriteOnce] |No      |The access mode should be use with the persistent volumes   |
+>- Set the [pullSecret](./install-config.yaml#L35), You can get this secret from https://cloud.redhat.com/openshift/install/pull-secret`
+>- Set the [sshKey](./install-config.yaml#L36) (content of ~/.ssh/id_rsa.pub)
+
+### Run the installer 
+
+Start the installation using the install-config created in the folder `install_dir`
+
+```console
+$ openshift-install create cluster --dir=install_dir
+``` 
+### After installation is done
+
+When the all prompts are done the installer will create ${HOME}/.kubevirt/kubevirt-config.yaml
+containing all required information about the connection with Engine.
+The installation process will create a temporary VM which will trigger bootstrap VM
+for later create three masters nodes. The masters nodes will create all services and
+checks required. Finally, the cluster will create the three workers node.
```

Comment:
All this section is unclear to me - I don't see a reason at this point (end of installation) to list again the installation process (bootstrap -> masters -> workers). It should simply list how the user access the new cluster.

Reply:
Agree, I remove this section, leave only the description how to access to the tenantcluster